tests/unit/s2n_record_read_test.c

```diff
@@ -198,9 +206,13 @@ int main(int argc, char *argv[])
 
         /* Invalidate an encrypted byte to cause decryption to fail. */
         struct s2n_stuffer invalidation_stuffer = stuffer_pair.server_in;
-        uint8_t *first_byte = s2n_stuffer_raw_read(&invalidation_stuffer, 1);
-        EXPECT_NOT_NULL(first_byte);
-        *first_byte += 1;
+        /* Offset and corrupt the 6th byte, which is used by both TLS 1.2 and TLS 1.3
+         * during decryption
+         */
```

Comment:
Would the last byte be simpler / more obvious?

Reply:
Skipping the last byte does work but I prefer skipping the header len amount since that is the reason why this currently doesnt work.



```
// skipping last byte tested via

int last_byte = s2n_stuffer_data_available(&invalidation_stuffer) - 1;    
...
EXPECT_SUCCESS(s2n_stuffer_skip_read(&invalidation_stuffer, last_byte)); 
```